### ISSUE TITLE
fMRI Biopac data

### ISSUE THREAD

**Issue by vimazz** [cliffordlab/PhysioNet-Cardiovascular-Signal-Toolbox#33]
Hello!
Is it possible to analyse fMRI biopac data (ECG and PPG) with this toolbox?
I have imported in .mat the .acq ECG but, for instance, I cannot run the BasicDemo with my data since I've not the .hea file. I tried to adapt it but it doesn't run (error in readheader).
Any suggestion?
Thanks in advance!

**Comment by GiuliaDAP** [comment]
Yes, you can use data collected with any device and independently from the format.

"_The toolbox does not assume any format of data except that the input of the Main_HRV_Analysis.m fucntion are a two equal length vectors: RR interval and time in units of seconds or the 'raw' ECG signal (physical units,mV) and time. Additionaly, blood pressure waveform and photoplethysmographic/pulsatile data can be analyzed and they should be in the standard physical units (mmHg or normalized units respectively)._"

Please see: https://github.com/cliffordlab/PhysioNet-Cardiovascular-Signal-Toolbox#i-getting-started
